src/maker/server.rs

```diff
@@ -464,12 +468,12 @@ fn handle_client(maker: Arc<Maker>, stream: &mut TcpStream) -> Result<(), MakerE
 ///
 /// The server continues to run until a shutdown signal is detected, at which point
 /// it performs cleanup tasks, such as saving wallet data and terminating active Tor sessions.
-pub fn start_maker_server(maker: Arc<Maker>) -> Result<(), MakerError> {
+pub fn start_maker_server(maker: Arc<Maker>, feerate: Option<f64>) -> Result<(), MakerError> {
```

Comment:
Instead of passing this value all the way from the top function, we can just use the default feerate 3 sats/vb for all fidelity bonds, creation and redemption. 

That will simplify the API. Also later on we will use more sophisticated fee estimation mechanism so the top level APIs don't need to change.